frontend/react/style-guide.md

```diff
@@ -594,29 +588,28 @@ src
 
   ```jsx
   function HOC(WrappedComponent) {
-    return class Proxy extends Component {
-      Proxy.propTypes = {
-        text: PropTypes.string,
-        isLoading: PropTypes.bool
-      };
-
-      render() {
-        return <WrappedComponent {...this.props} />
-      }
+    function Proxy(props) {
+      return <WrappedComponent {...props} />
     }
+
+    Proxy.propTypes = {
+      text: PropTypes.string,
+      isLoading: PropTypes.bool
+    };
```

Comment:
we dont use propType, we use TS, but this can be a task to do, uptade the style guide to TS.